code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -104,6 +119,11 @@
 	. = ..()
 	pull_vines()
 
+/mob/living/simple_animal/hostile/venus_human_trap/Moved(atom/OldLoc, Dir)
+	. = ..()
+	var/north_or_west = dir == NORTH || dir == WEST
+	pixel_x = north_or_west ? 2 : -2
```

Comment:
```suggestion
	pixel_x = dir & (NORTH|WEST) ? 2 : -2
```
Directions are bitflags, so we can use bitwise operations.
This will return a truthy value if it matches with either north or west (or north-and-west-including diagonals, but that's not a set of valid directions).